### PR TITLE
add in a title

### LINKED ISSUE #286: Need a decent title
The <title> for the management interface is just "Management", it should be something related to the function that the page is on.

### PR DESCRIPTION
**Pull request by andymckay**
* super simple setTitle as discussed, will probably be removed later when the data structure for pages lands
* remove prefix for now as per comments
* fixes #286

### CODE REVIEW

File: public/js/utils.jsx [hunk 1]
```diff
@@ -41,6 +41,11 @@ export function gettext(string){
 }
 
 
+export function setTitle(string) {
+  document.title = gettext('Pay') + ' | ' + string;
```
Comment (muffinresearch) [inline public/js/utils.jsx]: If we want to be consistent with FxA then we could totally drop the prefix.

File: public/js/views/management/history.jsx [hunk 2]
```diff
@@ -5,6 +5,10 @@ import { gettext } from 'utils';
 
 export default class History extends Component {
 
+  componentDidMount() {
+    setTitle(gettitle('My account'));
```
Comment (muffinresearch) [inline public/js/views/management/history.jsx]: This should be "Receipts and history"

File: public/js/utils.jsx [hunk 3]
```diff
@@ -41,6 +41,11 @@ export function gettext(string){
 }
 
 
+export function setTitle(string) {
```
Comment (muffinresearch) [inline public/js/utils.jsx]: Perhaps add a basic test for this. Maybe inject the document object that way you can check the property is set. You can now use default args `setTitle(string, doc=document)`.

File: public/js/views/management/history.jsx (hunk 2 above)
Comment (muffinresearch) [inline public/js/views/management/history.jsx]: Presumably `gettitle` shoudl be `gettext`.

Comment (muffinresearch) [inline public/js/views/management/history.jsx]: You're missing the import of `setTitle` here.

File: public/js/utils.jsx (hunk 1 above)
Comment (andymckay) [inline public/js/utils.jsx]: Marketplace does the pipe but does it as a suffix. Personally I'm not tied to it, blah lets remove it.

File: tests/test.utils.jsx [hunk 4]
```diff
@@ -71,3 +71,13 @@ describe('utils.isDisabled', function() {
   });
 
 });
+
+
+describe('utils.setTitle', function() {
+
+    it('should set the title', function() {
+        utils.setTitle('Pay', document);
```
Comment (muffinresearch) [inline tests/test.utils.jsx]: This is where you pass in your own object so that you don't manipulate the real title.

**Comment by muffinresearch** [comment]
r+wc

File: public/js/utils.jsx [hunk 5]
```diff
@@ -41,6 +41,11 @@ export function gettext(string){
 }
 
 
+export function setTitle(string, doc=document) {
+  doc.title = string;
```
Comment (muffinresearch) [inline public/js/utils.jsx]: I like this, I think it makes it simple e.g. no prefix/suffix or pipe. :+1:

**Pull request merged**